flake.lock

```diff
@@ -1865,11 +1865,11 @@
     "starknet-devnet-src": {
       "flake": false,
       "locked": {
-        "lastModified": 1732210861,
-        "narHash": "sha256-8PT1hGYu6AuF8vfzgGja0R9ZpIwgJxSLoqCGSqyJtxQ=",
+        "lastModified": 1737710043,
+        "narHash": "sha256-/XOsWJnA7Qt4Qmy2bACt4iGfynsJlIsh61nVyWDL4vc=",
         "owner": "0xSpaceShard",
         "repo": "starknet-devnet-rs",
-        "rev": "038d94612c6b6e51f06dc124bb320085485f588d",
+        "rev": "d2d5f6f13e58aadb41fa10de70ee782d177b62c6",
```

Comment:
ah. you updated this :sweat_smile: that's why it triggered the recompilation. do we need all of these changes?

Reply:
Not sure if we can update only the Cosmos Nix with `nix flake update`